### PR TITLE
add chain2paf

### PR DESCRIPTION
**Pull request by AndreaGuarracino**
Add `chain2paf` to convert sequence alignment from CHAIN to PAF format.

### CODE REVIEW

**Review by mbhall88 — CHANGES_REQUESTED** [review]
Please see the [guidelines for Rust recipes](https://bioconda.github.io/contributor/guidelines.html#rust) for licensing and additional cargo flags

**Comment by AndreaGuarracino** [comment]
Hi @mbhall88, sorry for the dumbness, what's missing about the LICENSE or flags? Thanks to @martin-g, also ARM's tests pass now.

**Comment by mbhall88** [comment]
No need to apologise. Sorry for my abrupt response. That link above has an example of a best practice recipe. In particular, bundling licenses of dependencies and adding no track option. Sorry, not at my computer at the moment to add code suggestions

File: recipes/chain2paf/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  number: 0
+  run_exports:
+      - {{ pin_subpackage(name, max_pin='x.x') }}
+  script:
+      - cargo-bundle-licenses --format yaml --output THIRDPARTY.yml
+      - cargo install -v --locked --no-track --root $PREFIX --path .
+
+requirements:
+  build:
+    - {{ compiler('rust') }}
```
Comment (mbhall88) [inline recipes/chain2paf/meta.yaml:22]:
```suggestion
    - {{ compiler('rust') }}
    - cargo-bundle-licenses
```

File: recipes/chain2paf/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+test:
+  commands:
+    - chain2paf --help
+
+about:
+  home: https://github.com/AndreaGuarracino/{{ name }}
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
```
Comment (mbhall88) [inline recipes/chain2paf/meta.yaml:39]:
```suggestion
  license_file: 
    - LICENSE
    - THIRDPARTY.yml
```

**Review by mbhall88 — APPROVED** [review]

**Pull request merged**